packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/AddStudentsToQueueModal.tsx

```diff
@@ -181,13 +189,19 @@ const AddStudentsToQueueModal: React.FC<AddStudentsToQueueModalProps> = ({
             <Input.TextArea autoSize={{ minRows: 3, maxRows: 6 }} allowClear />
           </Form.Item>
 
-          {/* TODO: change this to only be an option if the queue is hybrid. Strictly in-person or online queues should not have this option */}
-          <Form.Item name="location" label="Is the question in-person?">
-            <Radio.Group className="mb-1">
-              <Radio value="In Person">Yes</Radio>
-              <Radio value="Online">No</Radio>
-            </Radio.Group>
-          </Form.Item>
+          {isQueueHybrid && (
+            <Form.Item
+              name="location"
```

Comment:
You will want to add the following to the <Form> element, otherwise location will be null if the user doesn't click anything
```
initialValues={{
            location: isQueueHybrid ? 'Online' : undefined,
          }}
```
![image](https://github.com/user-attachments/assets/f6fca718-8309-48d1-8926-bd0c5ab870ff)

Reply:
also update with this, i realised I was on an old version, hence why it says [null] instead of [unselected]. It seems the issue is still there though